lib/src/extras/logging.rs

```diff
@@ -11,11 +13,11 @@ use tracing_log::NormalizeEvent;
 use tracing_subscriber::filter::Targets;
 use tracing_subscriber::fmt::format::Writer;
 use tracing_subscriber::fmt::time::{FormatTime, SystemTime};
-use tracing_subscriber::fmt::{FmtContext, FormatEvent, FormatFields, FormattedFields, Layer};
+use tracing_subscriber::fmt::{FmtContext, FormatEvent, FormatFields, FormattedFields};
 use tracing_subscriber::layer::SubscriberExt;
 use tracing_subscriber::registry::LookupSpan;
 use tracing_subscriber::util::SubscriberInitExt;
-use tracing_subscriber::Layer as _;
+use tracing_subscriber::Layer;
```

Comment:
And also this one related to `NIMIQ_MODULES`

Reply:
Already done in an earlier PR.